portal/src/texts/blog/iePlussJokulErlikSant/iePlussJokulErlikSant.mdx

```diff
@@ -5,9 +5,13 @@ author: Jo Emil Holen
 publishDate: "2020.08.05"
 ---
 
-"Vi må støtte IE 11" trenger ikke være den verste setningen en utvikler kan høre. Med noen enkle steg kan man komme veldig langt uten å hverken måtte snu opp-ned på markup eller kode.
+<Ingress>
+    "Vi må støtte IE 11" trenger ikke være den verste setningen en utvikler kan høre. Med noen enkle steg kan man komme
+    veldig langt uten å hverken måtte snu opp-ned på markup eller kode.
+</Ingress>
 
-*Oppdatert 2020.09.03 for NanoID.*
+-   Oppdatert 2021.11.11: [IE-støtte droppes i Jøkul](/blog/jokul-dropper-ie-stotte/).
```

Comment:
@joms jeg lister opp en endring her og lenker til posten. Men du tenkte deg kanskje en `WarningMessageBox` eller noe sånt?

Reply:
Ja, stemmer. Bare for ekstra visuell feedback om at denne posten ikke er viktig lenger med mindre man absolutt vil.